src/styles/styles.ts

```diff
@@ -1367,7 +1367,6 @@ const styles = (theme: ThemeColors) =>
         },
 
         sidebarListContainer: {
-            scrollbarWidth: 'none',
```

Comment:
On staging, we don't have a scroll bar in LHN.
Let's remove the scrollbar.

<img width="271" alt="image" src="https://github.com/Expensify/App/assets/29673073/6b9583d8-1295-4bd3-b9b2-30750b8b23db">